addons/survey/models/survey_survey.py

```diff
@@ -170,7 +170,7 @@ def copy_data(self, default=None):
         return super(Survey, self).copy_data(default)
 
     @api.multi
-    def _create_answer(self, user=False, partner=False, email=False, test_entry=False, **additional_vals):
+    def _create_answer(self, user=False, partner=False, email=False, test_entry=False, invite_token=False, **additional_vals):
```

Comment:
Not given `invite_token` implies we are in the "start a new pool" case. So, don't need it to check if attempt limit is reached. The invite_token will be given with default method of field definition.

Reply:
Not necessary anymore (as discussed).